src/Processor/AuthorisationProcessor.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Webhook Module for PHP
+ *
+ * Copyright (c) 2021 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Webhook\Processor;
+
+use Adyen\Webhook\EventCodes;
+use Adyen\Webhook\PaymentStates;
+
+class AuthorisationProcessor extends Processor implements ProcessorInterface
+{
+    public function process(): ?string
```

Comment:
I think a phpdoc would be a great addition for this function with the exception type